src/components/classic/releases/new-features-in-517.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default_md
+title: New Features in 5.17 (Work in Progress)
+title-class: page-title-activemq5
+type: activemq5
+---
+
+[Features](/features) > [New Features](/new-features) > [New Features in 5.17](components/classic/releases/new-features-in-517)
+
+__*This release is still a work in progress.*__
+
+New Features in 5.17
```

Comment:
Let's label this 'Work in Progress' here.. we can then move items that have merged PR's to the "New" or "Removed" sections below. From a work flow perspective, we can have a convention where WIP is something that has a PR, New/Removed Features is for merged PRs and Planned is JIRA only at the given time period.

Work in Progress:
- item a
- item b

New Features completed:
- item c
- tem d

Removed Features completed:
- item e

Planned features:
- item f
- item g

then the upgrading.. etc below

Reply:
I don't think anybody really wants to go and update the website anytime they complete a task tbh. It's pretty easy just to look at the jira if you want more detail about what's specifically in progress.

I think this is clear enough as is?

Reply:
I prefer 10x the word wip than release. Agreeing with Matt here.

Reply:
I quite like Kafkas approach where they have kips but a kip is never in the release unless its fully merged to main. Also there is nothing on the core site about kip stuff, it only has release information this avoids confusion to end users. Kips are only tracked in seperate dev wiki and jira, as its only dev until its released.